src/client/app/appActions.js

```diff
@@ -84,3 +84,11 @@ export const getCryptoPriceHistory = (symbol, refresh = false) => dispatch => {
     },
   });
 };
+
+export const SHOW_POST_MODAL = '@app/SHOW_POST_MODAL';
+export const HIDE_POST_MODAL = '@app/HIDE_POST_MODAL';
+
+export const showPostModal = createAction(SHOW_POST_MODAL);
+export const hidePostModal = createAction(HIDE_POST_MODAL);
+
+export const LOCATION_CHANGE = '@@router/LOCATION_CHANGE';
```

Comment:
We don't have to hard code this, it can be imported from react-router:
```js
import { LOCATION_CHANGE } from 'react-router-redux/reducer';
```

Reply:
oh didnt know that, thats cool will update